common/src/main/java/gov/cms/ab2d/common/model/CoveragePeriod.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.ab2d.common.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+import static javax.persistence.EnumType.STRING;
+
+@Entity(name = "bene_coverage_period")
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
+public class CoveragePeriod extends TimestampBase {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @EqualsAndHashCode.Include
+    private Integer id;
+
+    @ManyToOne
+    @JoinColumn(name = "contract_id")
+    private Contract contract;
+
+    @Column
+    private int month;
+
+    @Column
+    private int year;
+
+    @Enumerated(STRING)
+    private JobStatus status;
```

Comment:
What's jobstatus in here for?

Reply:
A contract beneficiary mapping search is now a job in an almost equivalent sense to the Job class. A mapping search is submitted, in progress, successful/failed. We need to know this because we don't want to begin a regular EOB search job when the mapping is being updated.